Gems/Atom/RHI/Vulkan/Code/Source/Platform/Linux/Vulkan_Traits_Linux.h

```diff
@@ -7,9 +7,9 @@
  */
 #pragma once
 
-#define AZ_TRAIT_ATOM_SHADERBUILDER_DXC "Builders/DirectXShaderCompilerAz/bin/dxc"
+#define AZ_TRAIT_ATOM_SHADERBUILDER_DXC "Builders/DirectXShaderCompiler/bin/dxc"
 #define AZ_TRAIT_ATOM_VULKAN_DISABLE_DUAL_SOURCE_BLENDING 0
-#define AZ_TRAIT_ATOM_VULKAN_DLL ""
-#define AZ_TRAIT_ATOM_VULKAN_DLL_1 ""
+#define AZ_TRAIT_ATOM_VULKAN_DLL "libvulkan.so"
+#define AZ_TRAIT_ATOM_VULKAN_DLL_1 "libvulkan.so.1"
 #define AZ_TRAIT_ATOM_VULKAN_LAYER_LUNARG_STD_VALIDATION_SUPPORT 0
```

Comment:
Validation layers are working for me, so we could activate them here.

Reply:
+1

Reply:
Will update to 1